src/main/java/service/impl/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,20 @@
+package service.impl;
+
+import model.FruitTransaction;
+import service.FruitTransactionParser;
+
+public class FruitTransactionParserImpl implements FruitTransactionParser {
+    @Override
+    public FruitTransaction parse(String line) {
+        String[] parts = line.split(",");
+        FruitTransaction.Operation operation = FruitTransaction.Operation.fromCode(parts[0]);
+        String fruit = parts[1];
+        int quantity = Integer.parseInt(parts[2]);

```

Comment:
It's recommended to handle `NumberFormatException` which could be thrown if `parts[2]` is not a valid integer.